src/mlpack/methods/lsh/lsh_main.cpp

```diff
@@ -64,6 +64,8 @@ PARAM_INT("tables", "The number of hash tables to be used.", "L", 30);
 PARAM_DOUBLE("hash_width", "The hash width for the first-level hashing in the "
     "LSH preprocessing. By default, the LSH class automatically estimates a "
     "hash width for its use.", "H", 0.0);
+PARAM_INT("num_probes", "Number of additional probes for Multiprobe LSH"
```

Comment:
We should probably add a semicolon after "LSH" and decapitalize the following "If" so it is one complete sentence. Or something like that.